packages/fast-development-site-react/src/components/site/index.tsx

```diff
@@ -745,7 +781,17 @@ class Site extends React.Component<ISiteProps & IManagedClasses<ISiteManagedClas
         });
     }
 
-    private handleLocaleUpdate = (e: any): void => {
+    private renderThemes(): JSX.Element[] {
+        return this.props.themes.map((theme: ITheme, index: number): JSX.Element => {
+            return (
+                <option key={index}>
```

Comment:
You can use the id here as they key and also as the value so this should be `<option key={theme.id} value={theme.id}>`

Reply:
For reference: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/option